src/test/scala/me/rexim/morganey/AutocompletionSpec.scala

```diff
@@ -16,7 +16,7 @@ class AutocompletionSpec extends FlatSpec with Matchers with TestTerms  {
   def autocomplete(line: String, cursor: Int, knownNames: List[String]): Set[CharSequence] = {
     val id = I(lvar("x"))
     val fakeBindings = knownNames.map(name => MorganeyBinding(lvar(name), id))
-    val moduleFinder = new ModuleFinder(List(new File("./std/")))
+    val moduleFinder = new ModuleFinder(List(new File("./test-data/load-autocomplete/")))
```

Comment:
Which actually brings up another interesting question: do we want the std-lib to be resource for the uberjar? #45 Which is indeed a separate question and can be discussed in #45 if needed. Simply wanted to mentioned it here.